charts/istio-authorizer/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: istio-authorizer
 description: A Helm chart for istio-authorizer
 type: application
-version: 1.17.0
-appVersion: "1.17.0"
```

Comment:
istio authorizer is in the same version as acp so this (appVersion) shouldn't be changed.